### PR TITLE
Cole second additions

### PR DESCRIPTION
**Pull request by bandaloo**
Added better images of the screenshots from the Adobe XD project and added a brief explanation about CSS

### CODE REVIEW

File: references.bib [hunk 1]
```diff
@@ -498,7 +498,7 @@ @online{yoo2017
 	author={Aaron Yoo},
 	title={iPhone X},
 	year={2017},
-	month=dec,
+    month=dec,
```
Comment (jojonium) [inline references.bib:501]: Use tabs like the rest of this file. Fix this in the other places spaces are used too

File: references.bib [hunk 2]
```diff
@@ -514,6 +514,9 @@ @online{boland2018
 	urldate={2019-09-27}
 }
 
+@comment{
+    weird broken line in this thing? does that matter?
```
Comment (jojonium) [inline references.bib:518]: Looks fine in the pdf

File: references.bib [hunk 3]
```diff
@@ -659,10 +662,22 @@ @online{kyotovr2018
 	urldate={2019-10-01}
 }
 
+@comment {
+    are images from wikimedia commons okay?
```
Comment (jojonium) [inline references.bib]: Yes as long as they have a permissive license. If it's Creative Commons you need to mention the license and provide a link to it.
`note={Licensed under \href{https://creativecommons.org/licenses/by/2.0/}{CC BY 2.0}},`

**Review by jojonium — CHANGES_REQUESTED** [review]
Make sure all images that aren't ours are cited and mention the license if necessary

File: references.bib (hunk 1 above)
Comment (bandaloo) [reply references.bib:501]: but people who use spaces instead make more money

**Pull request merged**